src/hotspot/cpu/ppc/continuationHelper_ppc.inline.hpp

```diff
@@ -43,11 +43,11 @@ inline intptr_t* ContinuationHelper::frame_align_pointer(intptr_t* p) {
 
 template<typename FKind>
 inline void ContinuationHelper::update_register_map(const frame& f, RegisterMap* map) {
-  Unimplemented();
+  // Nothing to do
 }
 
 inline void ContinuationHelper::update_register_map_with_callee(const frame& f, RegisterMap* map) {
-  Unimplemented();
+  // Nothing to do
```

Comment:
Would it be better to call the empty `frame::update_map_with_saved_link` to be consistent with the other platforms? @reinrich: You may have an opinion.

Reply:
I'd prefer this version. It is clearer about the fact that `map` doesn't need to be updated on ppc with data from the callee `f`.

Reply:
Ok, I'm fine with it, but would prefer to have a comment describing this.

Reply:
Or even: `Currently all registers are considered to be volatile and saved in the caller (java) frame if needed`.